src/server/views/dashboard/users-list.njk

```diff
@@ -10,19 +10,19 @@
     {{ dashboardMacros.successBanner(successBannerTitle, messages.userUpdatedEmail + notificationMessage, messages.userUpdatedNotificationColour) }}
   {% endif %}
   <table class="govuk-table">
-    <caption class="govuk-table__caption govuk-table__caption--l">Users</caption>
+    <caption class="govuk-table__caption govuk-table__caption--l">List administrators</caption>
     <thead class="govuk-table__head">
       <tr class="govuk-table__row">
         <th scope="col" class="govuk-table__header">Email</th>
-        <th scope="col" class="govuk-table__header">Roles</th>
+        <th scope="col" class="govuk-table__header">Super admin</th>
         <th scope="col" class="govuk-table__header govuk-table__header--numeric"></th>
       </tr>
     </thead>
     <tbody class="govuk-table__body">
       {% for user in users %}
         <tr class="govuk-table__row">
           <th scope="row" class="govuk-table__header">{{ user.email }}</th>
-          <td class="govuk-table__cell">{{ user.jsonData.roles.join(", ") }}</td>
+          <td class="govuk-table__cell">{{ "Super admin" if user.jsonData.roles.includes("Administrator") else "" }}</td>
```

Comment:
I think it looks a little strange that the column heading is "Super admin", and the data is also "super admin".. is that right? should the header still be "Role"?